### ISSUE TITLE
make: error in OLE_get_header

### ISSUE THREAD

**Issue by andreas-wolf** [inflex/ripOLE#1]
cc -Wall -g -O2 -I. -Werror  -c ole.c
ole.c: In function ‘OLE_get_header’:
ole.c:568:6: error: variable ‘result’ set but not used [-Werror=unused-but-set-variable]
  int result = 0;
      ^
cc1: all warnings being treated as errors
Makefile:7: recipe for target 'ole.o' failed
make: *** [ole.o] Error 1

**Comment by inflex** [comment]
For the short term, just remove "-Werror" from the Makefile.

**Comment by andreas-wolf** [comment]
Yes. Sure. Thanks. The method code from OLE_get_header looked somewhat messed up, so I thought someone should have a look.

**Comment by inflex** [comment]
ja, I can see there's a few possibles to catch there.  I'll clean it up.

**Comment by jiripech** [comment]
Actually, I've just cleaned it up. Feel free to get it from the pull request #7 or my fork.
